lib/generators/rolemodel/modals/templates/app/views/shared/_custom_confirm.html.slim

```diff
@@ -0,0 +1,8 @@
+.modal#custom-confirm.flex.items-center.justify-center
```

Comment:
I'm thinking we could actually drop the "custom" prefix on this naming. Couldn't we just call this RolemodelConfirm? We were using a custom to distinguish from the Rails built-in, but now we are fully overriding